cmake/multi_image.cmake

```diff
@@ -56,6 +56,11 @@ function(add_child_image)
   set(oneValueArgs NAME SOURCE_DIR DOMAIN)
   cmake_parse_arguments(ACI "" "${oneValueArgs}" "" ${ARGN})
 
+  if (NOT CONFIG_PARTITION_MANAGER_ENABLED)
```

Comment:
This test is here to ensure that we don't get regression by changes in kconfig. If we get to this part of the code we must ensure that the kconfig value is set.